common/source/docs/common-camera-gimbal.rst

```diff
@@ -173,7 +164,7 @@ below:
 -  If you have jerky camera movement adjust the RC_Feel parameter to a
```

Comment:
the RC_FEEL parameter is gone now and replaced instead of ATC_INPUT_TC but I'm not sure this is good advice to adjust this.  Anyway, if we want to keep this message here then it should be, "If you have jerky camera movement adjust the ATC_INPUT_TC to a higher value such as 0.5 or 0.2"